### PR TITLE
MT3-524 Feature/use checkboxes from lbh frontend react

### PR DESCRIPTION
**Pull request by LBHELewis**

### CODE REVIEW

**Comment by LBHELewis** [comment]
@erbridge this is ready for review now please

File: steps/property-inspection/pets.tsx [hunk 1]
```diff
@@ -75,51 +75,73 @@ const step = {
             ) as React.ReactNode,
             checkboxes: [
               {
-                label: "Amphibian (e.g. frog, newt, salamander, toad)",
+                label: {
+                  children: "Amphibian (e.g. frog, newt, salamander, toad)"
+                },
```
Comment (erbridge) [inline steps/property-inspection/pets.tsx]: Again, we should be able to insulate from this by making our `Checkboxes` map `label: string` to `label.children`. Given that the tests pass, it's not **necessary**, though.

**Review by erbridge — DISMISSED** [review]
A comment on a way this could be simplified, but not necessary.

File: components/Checkboxes.tsx [hunk 2]
```diff
@@ -29,66 +27,29 @@ export const Checkboxes: React.FunctionComponent<CheckboxesProps> = props => {
...
-    <>
-      <Fieldset className="checkboxes" legend={legend}>
-        {checkboxes.map(
-          (checkbox): JSX.Element => {
-            const labelId = `${name}-${checkbox.value.replace(
-              /\s+/g,
-              "-"
-            )}-label`;
-            const inputId = `${name}-${checkbox.value.replace(/\s+/g, "-")}`;
+  for (let i = 0; i < checkboxes.length; i++) {
```
Comment (erbridge) [inline components/Checkboxes.tsx]: This should get the same treatment as radios.

Comment (LBHELewis) [reply components/Checkboxes.tsx]: This is updated and ready for review now, thanks.

**Review by erbridge — APPROVED** [review]
Nice! This is much neater.

**Pull request merged**